File: code2map/docs/examples/java/output/parts/UserManagementService_findAll.java
// code2map fragment (non-buildable)
// original: docs/examples/java/UserManagementService.java
// lines: 119-121
// symbol: UserManagementService#findAll
// notes: references java.util.ArrayList, java.util.HashMap, java.util.List, java.util.Map, java.util.Optional, java.util.regex.Pattern; calls users.values
    public List<User> findAll() {
        return new ArrayList<>(users.values());
    }

File: code2map/docs/examples/python/output/parts/UserManagementService_find_all.py
# code2map fragment (non-buildable)
# id: CD10
# original: docs/examples/python/user_management_service.py
# lines: 151-158
# symbol: UserManagementService#find_all
# notes: references dataclasses.dataclass, re, typing.Optional; calls list, self._users.values
    def find_all(self) -> list[User]:
        """
        全ユーザーを取得する

        Returns:
            全ユーザーのリスト
        """
        return list(self._users.values())
